### ISSUE TITLE
windows portable release

### ISSUE THREAD

**Issue by coder2754** [AntiMicroX/antimicrox#343]
### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by pktiuk** [comment]
Good idea, but currently we lack resources for that.

**Comment by YUKI67000** [comment]
I beg you on my knees to make a portable version of AntiMicrox one day !

Currently, because of the use that I make of it, I am obliged to use the old "antimicro" (without the X) even if it has bugs, and to distribute this old version to all my friends because that we ABSOLUTELY NEED the program to be "portable".

I absolutely love what you have created, thank you for your great work, you can't imagine how much this little piece of software has saved my life ! [Excuse my English but I'm French]

**Comment by pktiuk** [comment]
As a temporary solution, I could try to create `.zip` file which after unpacking would contain all the files needed to launch this app.

**Comment by YUKI67000** [comment]
Omg, yes, please, that would be really awesome !

**Comment by pktiuk** [comment]
Yes, it may be helpful, but currently I am focused on other issues and I am nor going to work on this.

**Comment by AntumDeluge** [comment]
> As a temporary solution, I could try to create `.zip` file which after unpacking would contain all the files needed to launch this app.

I would consider that a permanent solution. A zip package is all I want for a portable download.

I also am using the original AntiMicro because it has a portable zip available. Normally, I would just extract the installer contents using [Universal Extractor](https://github.com/Bioruebe/UniExtract2). But it does not support AntiMicroX's installer format [yet].

**Comment by AntumDeluge** [comment]
Since there is no official portable release currently, I extracted the contents of the 3.2.5 installer & put it in a [.zip package for download](https://github.com/AntumDeluge/AntiMicroX/releases/tag/3.2.5).

**Comment by AntumDeluge** [comment]
Created a true portable release here: https://github.com/AntumDeluge/AntiMicroX/releases

Forked from master branch at 1722f6d631a61c0147f4d8cd2529fa2f8bdcd814.
